IOS/SwiftSeedProject/SwiftSeedProject/Application/Service/SourceService.swift

```diff
@@ -7,10 +7,31 @@
 //
 
 import Foundation
+import SwiftyJSON
 
 final class SourceService: BaseService<Source> {
     
     public override func updateLocalStoreWithServerInfo() {
-        //TODO: We should add the API CALL HERE
+        api.getSources(language: Language.English) { [weak self] result in
+            do {
+                guard let strongSelf = self else {
+                    return
+                }
+                let json = try result.unwrap()
+                let sources = json["sources"].arrayValue
+                sources.forEach { json in
+                    let id = json["id"].stringValue
+                    if let source = strongSelf.getEntityBy(identifier: id) as Source? {
+                        source.updateWithJSON(json: json)
```

Comment:
Mmm.. about these lines:

`source.updateWithJSON(json: json)`
`strongSelf.createEntity(json: json)`

The call to `updateWithJSON` will update the dabatase right-away? (or only in-memory?) If that's not the case, then maybe we should delegate the update into another class (the same that defines `createEntity`).